src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [response, setResponse] = useState({});
+  const [score, setScore] = useState(0);
+  const [displayScore, setDisplayScore] = useState(false);
+
+  const handleResponse = e => {
+    const selectedAnswer = { [e.target.name]: e.target.value };
+    setResponse({ ...response, ...selectedAnswer });
+  };
+
+  const checkAnswers = e => {
+    e.preventDefault();
+    const handleScore = data
+      .map(question => {
+        const responseAnswerId = response[question.id];
+        const responseObj = question.answers.find(
```

Comment:
I feel `responseObj` doesn't describe this well. What we're getting here is the `answer` object that the user selected, so a more descriptive name could be `selectedAnswer`

Reply:
I have already used `selectedAnswer` name on line 11, so it will be duplicate. How do you think  if I change the  `responseObj` name to `selectedAnswerObeject`instead

Reply:
You can still call it `selectedAnswer` even though the same name has been used in line 11 because we are in a different scope now. `selectedAnswer` on line 11 was created in the `handleResponse` function scope, and this one will be created in this `map` function scope which is absolutely fine in JS